src/commons/cluster_health.py

```diff
@@ -30,31 +30,33 @@
 LOGGER = logging.getLogger(ROOT)
 
 
-def check_health():
-    """K8s based health check."""
+def check_health(return_dict=None):
+    """K8s based health check.
+    :param return_dict : dictionary to share data between parallel processes.
+    """
     host, user, password = None, None, None
     for node in CLUSTER_CFG.nodes:
         if node["node_type"] == "master":
             host, user, password = node["hostname"], node["username"], node["password"]
             break
-    if not host:
-        raise HealthCheckError(f"Incorrect master details: {CLUSTER_CFG.nodes[0]} ")
-    cluster_obj = ClusterServices(host, user, password)
-    status, response = cluster_obj.check_cluster_health()
-    if not status:
-        raise HealthCheckError(f"Cluster is not healthy. response: {response}")
-    status, response = cluster_obj.check_cluster_storage()
-    if not status:
-        raise HealthCheckError(f"Failed to get cluster storage. response: {response}")
-    LOGGER.info(response)
-    LOGGER.info("Cluster is healthy, all services are up and running.")
-
-
-def health_check_process(interval):
+        if not host:
+            raise HealthCheckError(f"Incorrect master details: {CLUSTER_CFG.nodes[0]} ")
+        cluster_obj = ClusterServices(host, user, password)
+        status, response = cluster_obj.check_cluster_health()
+        if not status and not return_dict['is_deg_on']:
```

Comment:
We need specific check here instead skipping entire health check.

Reply:
> We need specific check here instead skipping entire health check.

@DhobaleVishal, this will require few little RND as how health check is currently implemented. May be need to update in next sprint.

Reply:
@VikasKumarC Please take note and create separate follow up task.